Rakefile

```diff
@@ -22,6 +21,7 @@ PuppetLint.configuration.log_format = "%{path}:%{linenumber}:%{check}:%{KIND}:%{
 PuppetLint.configuration.send('relative')
 PuppetLint.configuration.send('disable_80chars')
 PuppetLint.configuration.send('disable_class_inherits_from_params_class')
+PuppetLint.configuration.send('disable_arrow_alignment')
```

Comment:
Yeah, that's what the check does.  While all of your arrows are lined up now it helps ensure when someone else submits changes they continue to line up.  Also the forge uses this check in their effort to force syntax.  I think it should be enabled.